crates/client/rpc/src/madara_backend_client.rs

```diff
@@ -1,6 +1,7 @@
-use mc_db::DbError;
+use madara_runtime::opaque::{DBlockT, DHashT};
+use mc_db::{DbError, DeoxysBackend};
 use mc_rpc_core::utils::get_block_by_block_hash;
-use mp_block::Block;
+use mp_block::Block as MadaraBlock;
```

Comment:
please define it as DeoxysBlock instead

Reply:
Done! Usage of `DeoxysBlock`  has been refactored throughout the codebase: this includes uses of `mp_block::DeoxysBlock`.